tests/cypress/integration/masthead_test.spec.js

```diff
@@ -0,0 +1,69 @@
+import ListingPage from "../support/pages/admin_console/ListingPage.js";
+import LoginPage from "../support/pages/LoginPage.js";
+import SidebarPage from "../support/pages/admin_console/SidebarPage.js";
+import Masthead from "../support/pages/admin_console/Masthead.js";
+
+const loginPage = new LoginPage();
+const masthead = new Masthead();
+
+const logOutTest = () => {
+  it("logs out", () => {
+    masthead.signOut();
+    loginPage.isLogInPage();
+  });
+};
+
+const goToAcctMgtTest = (userDropdown) => {
+  it("opens manage account and returns to admin console", () => {
+    masthead.accountManagement();
+    cy.contains("Welcome to Keycloak Account Management");
+    cy.get("#landingReferrerLink").click({ force: true });
+    masthead.isAdminConsole();
+  });
+};
+
+describe("Masthead tests in desktop mode", () => {
+  beforeEach(() => {
+    cy.visit("");
+    loginPage.logIn();
+  });
+
+  goToAcctMgtTest("#user-dropdown");
+
+  it("disables header help and form field help", () => {
+    const sidebarPage = new SidebarPage();
+    const listingPage = new ListingPage();
+
+    sidebarPage.goToClientScopes();
+    listingPage.goToItemDetails("address");
+
+    cy.get("#view-header-subkey").should("exist");
+    cy.get(`#${CSS.escape("client-scopes-help:name")}`).should("exist");
+
+    masthead.toggleGlobalHelp();
+
+    cy.get("#view-header-subkey").should("not.exist");
+    cy.get(`#${CSS.escape("client-scopes-help:name")}`).should("not.exist");
```

Comment:
To have this as a reusable method (it could be used in tests to check that all the windows that contain help can show and hide it. It could be in `Masthead` as `helpIsShown(boolean)` where the "exist" or ''not.exist" change according to the boolean value
```suggestion
   masthead.helpIsShown(true);

    masthead.toggleGlobalHelp();

    masthead.helpIsShown(false);
```

Reply:
We should never hide the assertions in a utility class like Masthead.  You should always be able to read the test and know what it is asserting.  If I put this stuff into Masthead it would hide the entire test.

Also, we don't want to refactor something just because we think we might need it later.  It's extra complexity for no benefit.  Follow the [YAGNI principle](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).